src/Controller/PressPacksController.php

```diff
@@ -63,7 +63,7 @@ private function createFirstPage(array $arguments) : Response
             new Paragraph('Enquiries about published papers, material in press and eLife may be directed to <a href="mailto:press@elifesciences.org">press@elifesciences.org</a>, or +44&nbsp;1223&nbsp;855373.'),
             new Paragraph('If you’re writing about an eLife study, please cite eLife as the source of the article and include a link to either the article or elifesciences.org, preferably using our DOI: https://doi.org/10.7554/eLife – with the article’s five-digit extension (e.g. https://doi.org/10.7554/eLife.00000). Thank you!'),
             new Paragraph('All content, unless otherwise stated, is available under a <a href="https://creativecommons.org/licenses/by/4.0">Creative Commons Attribution License (CC-BY)</a>. All are free to use and reuse the content provided the original source and authors are credited.'),
-            new Paragraph('The <a href="'.$this->generateDownloadLink('https://cdn.elifesciences.org/downloads/eLife-media-policy-For-the-Press.pdf').'">eLife media policy</a> encourages authors to present and discuss their works ahead of publication and indicates that eLife does not release content under embargo, except in exceptional circumstances.'),
+            new Paragraph('The <a href="/media-policy">eLife media policy</a> encourages authors to present and discuss their works ahead of publication and indicates that eLife does not release content under embargo, except in exceptional circumstances.'),
```

Comment:
Should use the router rather than hard-coding local paths, like https://github.com/elifesciences/journal/blob/8d044d7c6ec3689eee7c7c9e34e0b0d619bf43cc/src/Controller/AboutController.php#L50